package/gluon-mesh-vpn-wireguard/luasrc/lib/gluon/upgrade/405-wireguard

```diff
@@ -12,14 +14,16 @@ local function generate_wg_iface_name(peer)
 end
 
 local function add_peer(group, name, config)
+  local ip = iputil.mac_to_ip("fe80::/64", sysconfig.primary_mac)
```

Comment:
This will give you a `/128` prefix, I think?

Reply:
yes. It does not seem to hurt though.

Reply:
Then how does the kernel get a route for the other end? Without any route, it won't be able to talk to it.

Reply:
I have no idea why it does work. But it does.

Reply:
Ah, maybe because link local addresses always specify the device which makes the route kinda redundant. But I am not sure about that.